### PR TITLE
Update RPR texts

### PR DESCRIPTION
**Pull request by ayyaruq**
Fixing a typo in Soul Reaver, and making Soulsow more explicit in Harvest Moon.

### CODE REVIEW

File: src/parser/jobs/rpr/changelog.tsx [hunk 1]
```diff
@@ -3,6 +3,11 @@ import CONTRIBUTORS from 'data/CONTRIBUTORS'
 import React from 'react'
 
 export const changelog = [
+	{
+		date: new Date('2022-04-24'),
+		Changes: () => <>Reword Harvest Moon text to make Soulsow's use clearer.</>,
```
Comment (eatmorehufu) [inline src/parser/jobs/rpr/changelog.tsx]: Use clearer... what? :P

File: src/parser/jobs/rpr/modules/Reaver.tsx [hunk 2]
```diff
@@ -103,7 +103,7 @@ export class Reaver extends Analyser {
 			</Trans>,
 			tiers: SEVERITIES,
 			why: <Trans id ="rpr.reaver.suggestion.dropped.why">
-					You used lost <Plural value={this.droppedReavers} one="# stack" other="# stacks"/> over the course of the fight.
+					You used <Plural value={this.droppedReavers} one="# stack" other="# stacks"/> over the course of the fight.
```
Comment (eatmorehufu) [inline src/parser/jobs/rpr/modules/Reaver.tsx]: this should be "lost" right? or is that not what droppedReavers means?

**Review by eatmorehufu — CHANGES_REQUESTED** [review]

File: src/parser/jobs/rpr/modules/Reaver.tsx (hunk 2 above)
Comment (ayyaruq) [reply src/parser/jobs/rpr/modules/Reaver.tsx]: Reading this I thought maybe I was high or something when I wrote this but it turns out following the exact request was a bad idea and the user wanted the wrong word dropped, will resolve in a bit.

File: src/parser/jobs/rpr/changelog.tsx (hunk 1 above)
Comment (ayyaruq) [reply src/parser/jobs/rpr/changelog.tsx]: Like, to make the usage of Soulsow clearer. Maybe an Australianism, it makes sense to me. Will update.

**Review by AzariahAeroborn — APPROVED** [review]

**Pull request merged**